.github/workflows/release.yml

```diff
@@ -13,10 +13,10 @@ jobs:
       GIT_USER_EMAIL: behlp@unityfoundation.io
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-      - uses: gradle/wrapper-validation-action@v1
+      - uses: gradle/actions/wrapper-validation@v3
```

Comment:
The `gradle/actions/wrapper-validation` action replaces `gradle/wrapper-validation-action`.
Reference:
https://github.com/gradle/wrapper-validation-action